.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Do we need these `.vscode` files? can we gitignore them?

Reply:
They can be ignored. Accumulated junk from when I was trying to sort out debug issues. I'll get rid of them and gitignore them just in case.